tests/structures/test_assignment.py

```diff
@@ -69,7 +69,15 @@ def test_list_assignment(self):
             print(x)
             print(y)
             print(z)
-            """)
+        """)
+
+    def test_bad_list_assignment(self):
+        self.assertCodeExecution("""
+            try:
+                [x, y, z, a] = range(3)
+            except:
```

Comment:
If you are wondering why the exception is ignored, it's because VOC throws a StopIteration while Cpython throws a ValueError (this behavior is captured in SequenceTests and they are expected failures).

Reply:
Added the tests that catch the correct error in the next commit (as expected failures), so these can be removed when handling bad unpacking of sequences is implemented.